lib/Importer/TFLiteModelLoader.cpp

```diff
@@ -1233,6 +1236,9 @@ Error TFLiteModelLoader::loadUnaryArithmetic(const tflite::Operator *op,
   NodeValue output;
   if (opCode == tflite::BuiltinOperator_LOGISTIC) {
     output = F_->createSigmoid(opInfo.name, outTy, input);
+  } else if (opCode == tflite::BuiltinOperator_HARD_SWISH) {
+    return loadHardSwish(op, opInfo, input);
```

Comment:
Looks unnecessary to create the `loadHardSwish` function. You should remove that function and put here only:
```
output = F_->createHardSwish(opInfo.name, input);
```